docs/embeddable-uis/element/access-requests.mdx

```diff
@@ -0,0 +1,589 @@
+| `INVALID_PERMISSION_LEVEL` | This error will emerge when you are trying to access part of the Element which you have not been given access too. This usually means the role that the person obtained has remained in the **Hidden Roles** permission level.                     |
+| `FORBIDDEN_ACCESS`         | This error will emerge when you are trying to login to an Element you have not been given permission to see.                                                                                                                                       |
+
+## API requests
+
+To use the Access Requests API, follow these steps:
+
+First of all we need to [get our `API_SECRET_KEY`](/api/api-with-cli/#get-your-api-key) from the dashboard and [get the current `project_id` and `env_id`](/api/examples/get-project-and-env)<br/>
+Replace `API_SECRET_KEY` with the key from the Permit dashboard along with the `project_id` and `env_id` you got from the API in the following commands.
+
+### Creating an Access Request
+
+To create a new access request, make a POST request to the following with the required data.
+
+```bash
+curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/access_requests' \
+  -H 'element_id: ELEMENTS_CONFIG_ID'
+  -data-raw
+ {
+    "access_request_details": {
+        "tenant": "34f5c98e-f430-457b-a812-92637d0c6fd0",
+        "resource": "4d5215ed-38bb-48ed-879a-fdb9ca58522f",
+        "resource_instance": "2d98d9f8-e1b7-4f1d-baad-2edbf6fa6c66",
+        "role": "ac4e70c8-d5be-48af-93eb-760f58fc91a9",
+    },
+    "time_limit": {
+        "start_time": "2019-08-24T14:15:22Z",
+        "end_time": "2019-09-24T14:15:22Z",
+    },
+    "reason": "done onboarding",
+ } \
+```
+
+The return access request object will look like this:
+```json
+{
+    "requesting_user_id": "1c1e4ada-f282-40e6-b3b7-20b3a51c93b5",
+    "access_request_details": {
+        "tenant": "34f5c98e-f430-457b-a812-92637d0c6fd0",
+        "resource": "4d5215ed-38bb-48ed-879a-fdb9ca58522f",
+        "resource_instance": "2d98d9f8-e1b7-4f1d-baad-2edbf6fa6c66",
+        "role": "ac4e70c8-d5be-48af-93eb-760f58fc91a9",
+    },
+    "reason": "done onboarding",
+    "org_id": "7c60d51f-b44e-4682-87d6-449835ea4de6",
+    "project_id": "405d8375-3514-403b-8c43-83ae74cfe0e9",
+    "env_id": "40ef0e48-a11f-4963-a229-e396c9f7e7c4",
+    "created_at": "2019-08-24T14:15:22Z",
+    "updated_at": "2019-08-24T14:15:22Z",
+    "time_limit": {
+        "start_time": "2019-08-24T14:15:22Z",
+        "end_time": "2019-09-24T14:15:22Z",
+    },
+    "status": None,
+    "reviewer_user_id": None,
+    "reviewed_at": None,
+    "reviewer_comment": None,
+}
+```
+## Reviewer actions:
+
+### 1.Get an Access Request
+
+To get a specific access request, make a GET request to the following.
+```bash
+curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/access_requests/{access_request_id}' \
+  -H 'element_id: ELEMENTS_CONFIG_ID'
+```
```

Comment:
Same here, you are not using the cookie.

Reply:
fixed